backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -1584,8 +1584,31 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
         return !irClass.isFrozen
     }
 
+    private fun isDefaultValue(value: IrExpression): Boolean {
```

Comment:
The name of the function is misleading: default for what?
Maybe `isConstZeroValue`?